frontend/src/components/EndpointTesting.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react'
```

Comment:
is this just a test file? if so do we need it in PR

Reply:
Removed from PR